flow/connectors/core.go

```diff
@@ -255,7 +259,8 @@ var (
 	_ NormalizedTablesConnector = &connclickhouse.ClickhouseConnector{}
 
 	_ QRepPullConnector = &connpostgres.PostgresConnector{}
-	_ QRepPullConnector = &connsqlserver.SQLServerConnector{}
+	// _ QRepPullConnector = &connsqlserver.SQLServerConnector{}
```

Comment:
Need to implement GetTableSchema for SQLServerConnector

Or push to have support for it removed